pkg/publicapi/apimodels/auth.go

```diff
@@ -2,8 +2,11 @@ package apimodels
 
 import (
 	"github.com/bacalhau-project/bacalhau/pkg/authn"
+	"github.com/pkg/errors"
```

Comment:
```suggestion
	"errors"
```

We can use the standard library version of errors to call errors.New.